service/generate_season.sql

```diff
@@ -0,0 +1,113 @@
+SET time_zone='+00:00';
+
+SELECT @Seasonnumber := MAX(season_number) FROM league_season;
+SET @Seasonnumber = @Seasonnumber + 1;
```

Comment:
Could we parameterize by league name as well so there isn't quite as much copy pasta? And then in the code just call it for each league. Because right now if we wanted to add or remove a league then this SQL won't be correct anymore.

Reply:
I see your point, but I am considering changing the rating thresholds for the 1v1 league, because the general rating is lower there. As soon as that happens a parametrization per league becomes way less elegant.

Reply:
And we have to update the file as soon as we want to change something with the league setup anyway

Reply:
It just feels to me that this defeats the entire purpose of storing stuff as data in a database. It couples the code to the data inserted in the database and if they get out of sync you run into issues. I would be a lot happier if it could just copy the setup of the previous season instead of having so much hardcoded stuff. Then you can change the setup of future seasons by just inserting data manually, and the code knows how to handle it from there.

Reply:
We can try that. Though that would need to happen in an new PR. I don't have the time to do and test that refactor in the next days and we need to get this deployed as the current season will end in a few days. So I will make a release with this version and then we have time to improve the whole process later.